thefiletree.docker

```diff
@@ -0,0 +1,26 @@
+# Take the latest version of nodejs container.
+FROM google/nodejs
+MAINTAINER David Aparicio "david.aparicio@free.fr"
+
+# Install tree build dependencies.
+RUN apt-get -qy install libicu-dev #node-gyp dependencies
+
+# Don't be root.
+RUN useradd -m user
+USER user
+ENV HOME /home/user
+WORKDIR /home/user
+
+# Download Tree's source code.
+RUN git clone https://github.com/garden/tree.git
+WORKDIR tree
+
+# Build Tree.
+RUN make links   #make without auto-starting
+RUN npm install #install dependencies like camp
```

Comment:
Nit: Same as above. It's nice that you document things well, but I don't really like end-of-line comments. Thanks!